### PR TITLE
Create new Release Candidate 2.44.0-rc0

### PR DESCRIPTION
**Pull request by bboreham**
See CHANGELOG for contents.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 2.44.0-rc.0 / 2023-04-22
+
+* [CHANGE] remote-write: raise default samples per send to 2,000. #12203
+* [FEATURE] remote-read: handle native histograms. #12085, #12192
+* [FEATURE] promtool: health and readiness check of prometheus server in CLI. #12096
+* [FEATURE] promql engine: add query_samples_total metric, the total number of samples loaded by all queries. #12251
```
Comment (juliusv) [inline CHANGELOG.md:8]:
```suggestion
* [FEATURE] promql engine: add `query_samples_total` metric, the total number of samples loaded by all queries. #12251
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+* [ENHANCEMENT] scraping: reduce memory allocations on Target labels. #12084
```
Comment (juliusv) [inline CHANGELOG.md:9]: "Target" -> "target"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+* [ENHANCEMENT] promql: use faster heap method for topk/bottomk. #12190
```
Comment (juliusv) [inline CHANGELOG.md:10]:
```suggestion
* [ENHANCEMENT] promql: use faster heap method for `topk()` / `bottomk()`. #12190
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+* [ENHANCEMENT] rules API: Allow filtering by rule name. #12270
+* [ENHANCEMENT] native histograms: various fixes and improvements. #11687, #12264, #12272
+* [ENHANCEMENT] ui: search of "Scrape Pools" is now case-insensitive. #12207
```
Comment (juliusv) [inline CHANGELOG.md:13]: I realize the original PR had "Scrape Pools" capitalized and in quotes like that, but I don't think there's a reason to do that... I'd just say "Searching within scrape pools..."

**Comment by roidelapluie** [comment]
I think we should add the stringlabels tag in .promu.yml in this PR

**Comment by bboreham** [comment]
Thanks for reviews!

Updated to reflect Julius' and Julien's comments. 

Do you think some of the CI tests need to be changed to set `-tags stringlabels`?  We are running unit tests both ways, but what about Windows, older Go version, etc.?

**Comment by roidelapluie** [comment]
I don't think so. We will soon make it default anyway

**Review by roidelapluie — APPROVED** [review]
LGTM

**Review by Nexucis — APPROVED** [review]
🥳

**Review by beorn7 — APPROVED** [review]

**Comment by bboreham** [comment]
GitHub says "Build Prometheus for all architectures _Expected — Waiting for status to be reported_".
I will try re-running all CI jobs.

**Comment by roidelapluie** [comment]
This is because you named your branch release - it's fine as it.

**Comment by bboreham** [comment]
/prombench cancel

It's failing to build due to builder being on Go 1.18
Fix: https://github.com/prometheus/test-infra/pull/529

**Comment by codesome** [comment]
No PromQL queries are being run. Should we stop the benchmark and fix that first?

**Comment by bboreham** [comment]
/prombench cancel

I filed https://github.com/prometheus/test-infra/issues/532

**Comment by bboreham** [comment]
Looking good on the stats we did get:
image image

**Comment by codesome** [comment]
~25% less memory, nice!

**Comment by bboreham** [comment]
We have queries in prombench as of 90 minutes ago.

**Comment by bboreham** [comment]
I just restarted both Prometheus in the prombench run (via Google Cloud console), to check if there is any difference when reloading from WAL.

**Comment by bboreham** [comment]
/prombench cancel

Looking good - peak RSS 31.5 against 38.6 before, peak heap 27.9 against 34.2 before:

image

image

image

image

**Pull request merged**